common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/TokenErrorResponse.java

```diff
@@ -0,0 +1,38 @@
+package com.microsoft.identity.common.internal.providers.oauth2;
+
+import com.google.gson.annotations.SerializedName;
+
+public class TokenErrorResponse {
+
+    @SerializedName("error")
+    protected String mError;
```

Comment:
Nit: I prefer to format field declarations with a space following each one such that

```
@SerializedName("error")
protected String mError;

@SerializedName("error_description")
protected String mErrorDescription;
```